.github/workflows/gradle.yml

```diff
@@ -31,8 +31,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gradle-version: [ "5.6.4", "7.6.1"]
-        # Plugin not currently compatible with Gradle 6.9.x
+        # Test newest 5.x, 6.x and 7.x, plus oldest patched 7.x and 8.x
+        # Latest 8.x is tested in 'quick-check' job
+        gradle-version: [ "5.6.4", "6.9.4", "7.0.2", "7.6.1", "8.0.2"]
```

Comment:
I really appreciate your commitment to supporting this wide number of versions! 🎉